src/components/HomePage/HothDescription.js

```diff
@@ -100,7 +101,7 @@ export default function HothDescription() {
 						Hack on the Hill?
 					</Typography>
 					<Typography variant='h2' className={classes.catchphrase}
-						style={{ fontWeight: 500, paddingBottom: isSmall ? theme.spacing(2) : theme.spacing(8) }}>
+						style={{ fontWeight: 600, paddingBottom: isSmall ? theme.spacing(2) : theme.spacing(8) }}>
```

Comment:
Let's delete the `fontWeight` styling here and add `fontWeight: 400` to the `catchphrase` styling under`useStyles`